app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -838,18 +877,49 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
             // activity transition animation
             ActivityTransitionAnim.transition(ZulipActivity.this);
+        } else if (requestCode == REQUEST_PICK_FILE && resultCode == RESULT_OK) {
+            List<Uri> fileUris = new ArrayList<>();
+            if (data.getData() != null) {
+                fileUris.add(data.getData());
+            }
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
Is this fine, generally there is a if condition for the versions below 16 (JELLY BEAN) ? 
If yes add a comment here!